custom_components/tuya_local/devices/airquality6in1.yaml

```diff
@@ -0,0 +1,70 @@
+name: Air quality monitor 6 in 1
+products:
+  - id: ugjbnqj2mffaexp5
+    name: Air Housekeeper
+primary_entity:
+  entity: sensor
+  name: PM2.5
+  class: volatile_organic_compounds
+  icon: "mdi:factory"
+  dps:
+    - id: 2
+      type: integer
+      name: sensor
+      unit: µg/m³
+      class: measurement
+      mapping:
+        - scale: 1
+secondary_entities:
+  - entity: sensor
+    class: temperature
+    dps:
+      - id: 18
+        type: integer
+        name: sensor
+        unit: C
+        class: measurement
+        mapping:
+          - scale: 10
+  - entity: sensor
+    class: humidity
+    dps:
+      - id: 19
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+        mapping:
+          - scale: 10
+  - entity: sensor
+    name: Formaldehyde
+    class: volatile_organic_compounds
+    dps:
+      - id: 20
+        type: integer
+        name: sensor
+        unit: mg/m³
```

Comment:
According to https://www.home-assistant.io/integrations/sensor/, units should be µg/m³ (ugm3 can be used as a substitute in the config file), so I would remove the scaling to make it so.